src/test/java/org/ow2/petals/flowable/incoming/integration/UtilsTest.java

```diff
@@ -0,0 +1,116 @@
+/**
+ * Copyright (c) 2018 Linagora
```

Comment:
I think the year should be 2019.

Reply:
Sure, but we are on a branch whith coding rules based on 2018. Copyright date is moved to 2019 on branch 1.1.x